src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -197,86 +232,74 @@ function WebAuthnCard(props) {
-										</td>
-										<td className="p-2 td-not-display align-middle">
-											<DateTime value={obj?.last_login}/>
-										</td>
-										<td className="p-2 align-middle">
-											<Button
-												outline
-												size="sm"
-												color="danger"
-												title={t("WebAuthnScreen|Unregister authenticator")}
-												onClick={(e) => {confirmWebAuthnUnregister(obj?.id), e.preventDefault()}}
-												className="float-right"
-												disabled={isSubmitting}
-											>
-												{t("WebAuthnScreen|Unregister")}
-											</Button>
-										</td>
-									</tr>
-								)
-							})}
-						</tbody>
-					</Table>
-					:
+			<CardBody>
+				{authenticators && authenticators.length > 0 ?
+					<Form onSubmit={handleSubmit(onSubmitKeyName)}>
+						<Table responsive borderless>
+							<thead>
+								<tr>
+									<th className="webauthn-th-keyname">
+										{t('WebAuthnScreen|Key name')}
+									</th>
+									<th className="webauthn-th-sign-count td-not-display">
+										{t('WebAuthnScreen|Sign count')}
+									</th>
+									<th className="webauthn-th-last-login td-not-display">
+										{t('WebAuthnScreen|Last successful login')}
+									</th>
+									<td className="webauthn-th-button"></td>
+								</tr>
+							</thead>
+							<tbody>
+								{authenticators.map((obj, idx) => {
+									return (
+										<TableRow
+											key={idx}
+											obj={obj}
+											regName={regName}
+											setValue={setValue}
+											formState={formState}
+											getValues={getValues}
+											resetField={resetField}
+											isSubmitting={isSubmitting}
+											globalEditMode={globalEditMode}
+											setGlobalEditMode={setGlobalEditMode}
+											confirmWebAuthnUnregister={confirmWebAuthnUnregister}
+										/>
+									)
+								})}
+							</tbody>
+
+						</Table>
+					</Form>
+				:
 					isLoading ?
-						<CellContentLoader cols={1} rows={2} title={t("WebAuthnScreen|Loading")}/>
+							<CellContentLoader cols={1} rows={2} title={t("WebAuthnScreen|Loading")}/>
 						:
-						<p className="text-center">
-							{t("WebAuthnScreen|No authenticator registered")}
-						</p>
-					}
-					<div className="div-button-center">
-						<Button
-							block
-							className="button-webauthn-register justify-content-center"
-							color="primary"
-							type="button"
-							disabled={isSubmitting}
-							onClick={(e) => {onRegister(), setIsSubmitting(true), e.preventDefault()}}
-						>
-							{t("WebAuthnScreen|Register new authenticator")}
-						</Button>
-					</div>
-				</CardBody>
+							<p className="text-center">
+								{t("WebAuthnScreen|No authenticator registered")}
+							</p>
+				}
+				<div className="div-button-center">
+					<Button
+						block
+						className="button-webauthn-register justify-content-center"
+						color="primary"
+						type="button"
+						disabled={isSubmitting || globalEditMode == true}
```

Comment:
@Aringoaway `disabled={isSubmitting || (globalEditMode == true)}` you are missing parenthesis